src/components/cqlLibraryLanding/cqlLibraryListActionCenter/CqlLibraryListActionCenter.tsx

```diff
@@ -56,6 +56,13 @@ export function CqlLibraryListActionCenter(props: PropTypes) {
     });
   }
 
+  const shareLibrary = (option: string) => {
```

Comment:
this callback is being passed as prop to component. can wrap this in useCallback.